src/splitInteger.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger([]).constructor.name).toEqual('Array');
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(1, 1)).toEqual([1]);
+  });
+
+  it(`should return object`, () => {
+    expect(typeof splitInteger('input')).toEqual('object');
+  });
+
+  it(`should return array length`, () => {
+    expect(splitInteger(10, 5)).toHaveLength(5);
+  });
+
+  it(`should return 0 if divided by a number`, () => {
+    expect(splitInteger(21, 8)).toEqual([2, 2, 2, 3, 3, 3, 3, 3]);
+  });
+
+  it(`should return 0 if divided by a number`, () => {
```

Comment:
Text in messages in the same for all tests below (should return 0 if divided by a number).
Change messages in all tests (like `should return an array [4, 4, 4, 5] with num = 17 if parts = 4`)